lib/t/matches.ex

```diff
@@ -125,7 +138,16 @@ defmodule T.Matches do
         [user_id_1, user_id_2] = Enum.sort(user_ids)
         m = "New match: #{user_id_1} and #{user_id_2}"
         Bot.async_post_message(m)
-        Repo.insert(%Match{user_id_1: user_id_1, user_id_2: user_id_2})
+        {:ok, match} = Repo.insert(%Match{user_id_1: user_id_1, user_id_2: user_id_2})
+
+        Repo.insert(%MatchEvent{
+          timestamp: DateTime.utc_now() |> DateTime.truncate(:second),
+          match_id: match.id,
+          event: "created"
+        })
+
+        expiration_date = expiration_date(match.id)
+        {:ok, %Match{match | expiration_date: expiration_date}}
```

Comment:
`expiration_date` appears to be a view concern, try moving it to `TWeb.MatchView`